src/Backend/Core/Engine/Header.php

```diff
@@ -129,7 +91,7 @@ private function buildPathForModule(string $fileName, string $module, string $su
             return '/src/Backend/Core/' . $subDirectory . '/' . $fileName;
         }
 
-        return '/src/Backend/Modules/' . $module . '/' . $subDirectory . '/' . $fileName;
+        return '/src/Backend/Modules/' . $module . '/' . $subDirectory . '/' . 'index.js';
```

Comment:
I forgot to mention that this part needs to be removed. The changes were made in the beginning when there were different plans. I would like that someone of @forkcms/backenders removes this, because the public addJS function needs to change.